src/stcal/jump/jump.py

```diff
@@ -658,32 +673,30 @@ def extend_saturation(
             alpha = ellipse[2]
             axis1 = min(axis1, max_extended_radius)
             axis2 = min(axis2, max_extended_radius)
-            image = cv.ellipse(
-                image,
-                (round(ceny), round(cenx)),
-                (round(axis1 / 2), round(axis2 / 2)),
-                alpha,
-                0,
-                360,
-                (0, 0, 22),  # in the RGB cube, set blue plane pixels of the ellipse to 22
-                -1,
+            ellipse_mask = skimage.draw.ellipse(
+                r=round(ceny),
+                c=round(cenx),
+                r_radius=round(axis1/2),
+                c_radius=round(axis2/2),
+                shape=image.shape,
+                rotation=alpha,
             )
+            image[*ellipse_mask, 2] = 22
```

Comment:
replaces `cv.ellipse` with a mask derived from `skimage.draw.ellipse`, then applies Blue 22 to the image array (why is the blue channel set to 22 if it's discarded right after? can this just be a simple mask?)

Reply:
No real reason. It's hack to create a mask of the ellipse. Sorry that's it is confusing.